cuda/base/cusparse_bindings.hpp

```diff
@@ -850,10 +850,6 @@ inline void destroy(cusparseSpMatDescr_t descr)
 #endif
 
 
-// CUDA versions 9.2 and above have csrsm2.
```

Comment:
So, we don't support CUDA 9.2 at all ? I thought this PR would stop using a CI job for 9.2 but we would still support 9.2 ? Maybe we can leave these #if defines, so if someone wants to compile with CUDA <9.2, they still can ?

Reply:
That would very much complicate #764 (due to the support for multiple rhs/need for tranposition), so I would really prefer to just deprecate support for older versions. Compute capabilities down to 3.0 are still officially supported in CUDA 10, so we are not losing any hardware support by this.

Reply:
Okay, then that is fine with me. Please also update the README.md, INSTALL.md and other pages where we currently say we support CUDA 9.0+. I guess now it would be CUDA 10.0+

Reply:
9.2 still works fine :)